core/storage/collection/gae_models.py

```diff
@@ -196,6 +196,35 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @staticmethod
+    def transform_dict_to_valid(model_dict):
+        """Replace invalid fields and values in the CollectionRightsModel dict.
+
+        Some old CollectionRightsSnapshotContentModels can contain fields
+        and field values that are no longer supported and would cause
+        an exception when we would try to create CollectionRightsModel. We need
```

Comment:
Strike "would"

Also change to: ... try to reconstitute a CollectionRightsModel from them. ...

Reply:
Done.